samples/javacv-demo-kotlin/settings.gradle.kts

```diff
@@ -0,0 +1,17 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * The settings file is used to specify which projects to include in your build.
+ *
+ * Detailed information about configuring a multi-project build in Gradle can be found
+ * in the user manual at https://docs.gradle.org/6.5/userguide/multi_project_builds.html
+ */
+
+pluginManagement {
+    repositories {
+        gradlePluginPortal()
+        maven { url = uri("https://oss.sonatype.org/content/repositories/snapshots/") }
```

Comment:
This isn't going to work for users building JavaCPP locally or using a release version.